fission/src/systems/scene/SceneRenderer.ts

```diff
@@ -219,6 +227,22 @@ class SceneRenderer extends WorldSystem {
         return screenSpace.unproject(this.mainCamera)
     }
 
+    /**
+     * Convert world space coordinates to screen space coordinates
+     *
+     * @param world World space coordinates
+     * @returns Pixel space coordinates
+     */
+    public WorldToPixelSpace(world: THREE.Vector3): PixelSpaceCoord {
```

Comment:
Do you want to support depth yet or leave it for later?

Reply:
I think leave depth for later